app/controllers/amr_heatmap_controller.rb

```diff
@@ -0,0 +1,39 @@
+class AmrHeatmapController < ApplicationController
+  include ReportHelper
```

Comment:
I don't think you need `ReportHelper` any more.

Reply:
Removed!